airbyte_cdk/test/standard_tests/connector_base.py

```diff
@@ -59,7 +58,16 @@ def connector(cls) -> type[IConnector] | Callable[[], IConnector] | None:
         try:
             module = importlib.import_module(expected_module_name)
         except ModuleNotFoundError as e:
-            raise ImportError(f"Could not import module '{expected_module_name}'.") from e
+            raise ImportError(
```

Comment:
@dbgold17 - Per our discussions earlier this week, this new longer error message goes deeper into the reasons that FAST doesn't find the module to import, and what can be done about it.